R/application.R

```diff
@@ -32,18 +32,26 @@ Application <- R6::R6Class(
     #' Print the object to the console
     #' @param ... Rest arguments.
     print = function(...) {
-      private$.printClass()
-      self$startTime$printValue()
-
-      if (!is.null(self$infusionTime)) {
-        self$infusionTime$printValue()
+      ospsuite.utils::ospPrintClass(self)
+      # Get the container of the application to get its path. Using the parameter
+      # of the application to get the parent container.
+      container <- self$startTime$parentContainer
+      while (container$containerType != "Application") {
+        container <- container$parentContainer
       }
-
-      if (!is.null(self$drugMass)) {
-        self$drugMass$printValue()
+      infusionTime <- NULL
+      if (!is.null(self$infusionTime)) {
+        infusionTime <- self$infusionTime$getPrintValue()
```

Comment:
`getPrintValue()` returns a string with user-friendly representation of the value.